src/styles/blocks/header.scss

```diff
@@ -0,0 +1,49 @@
+@use '../utils/mixins' as *;
+
+.header {
+  background-color: black;
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 100vh;
+  padding: 0 20px 80px;
+  box-sizing: border-box;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, 130px);
```

Comment:
on mobile and tablet it's better to have 1fr instead of 130px, because if your screen is wider than 260px so you have to adjust it to the width